src/components/Card/card.js

```diff
@@ -0,0 +1,29 @@
+import CounterBox from "../CounterBox/counterBox";
+import Modal from "react-bootstrap/Modal";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./card.css";
+
+const Card = (props) => {
```

Comment:
@PuniaMohit Can you read https://www.w3schools.com/react/react_es6_destructuring.asp and implement for this props?